src/subcommands/utils.go

```diff
@@ -168,6 +168,23 @@ func fetchFromCopy(indexPath string, name string) ([]byte, error) {
 	return []byte(ile.Content), nil
 }
 
+func createSearchDictionary(ile *pb.IndexListEntry) error {
```

Comment:
Why are we moving this into utils again? I think it should stay in lexical_search.go or go somewhere else dedicated to search

Reply:
createSearchDictionary function is called in the Add subcommand. And the word occurrences are written into the index list. So it's not exclusive for lexical search.

Reply:
Let's add it to a new file like src/search/some_file.go

Reply:
In general we should always avoid adding stuff to utils and what we do add should be purely simple one-off stuff, not core functionaltiy

Reply:
Adding a new file makes more sense. We might have to add more functions later as we add semantic search.